_data/developer.yml

```
@@ -307,7 +307,7 @@ websites:
     img: infobip.png
     tfa:
       - sms
```

Sounds like they support 2FA using both SMS and Phone calls.
```suggestion
      - sms
      - phone
```